core/build.gradle

```diff
@@ -19,8 +19,10 @@ apply plugin: 'biz.aQute.bnd.builder'
 
 dependencies {
   api project(':api')
-  api "org.terracotta:statistics:$parent.statisticVersion"
+  implementation "org.terracotta:statistics:$parent.statisticVersion"
+  implementation "org.terracotta.management:management-model:$terracottaPlatformVersion"
```

Comment:
This bothers me... for reasons I can't really elucidate.

Reply:
lets talk